.travis.yml

```diff
@@ -1,21 +1,23 @@
 language: ruby
 rvm:
-- 2.3.4
+  - 2.3.5
+  - jruby
 cache: bundler
 before_script:
-  - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+  - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64
+    > ./cc-test-reporter
   - chmod +x ./cc-test-reporter
-  - ./cc-test-reporter before-build
+  - "./cc-test-reporter before-build"
 after_script:
-  - if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then ./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT; fi
```

Comment:
(📓 CodeClimate's coverage reporter is able to handle that reports have already been submitted now, so we don't need this manual check any more).